lib/integrations/awsoidc/idp_iam_config.go

```diff
@@ -295,13 +162,12 @@ func NewIdPIAMConfigureClient(ctx context.Context) (IdPIAMConfigureClient, error
 		awsConfig:  cfg,
 		Client:     iam.NewFromConfig(cfg),
 		stsClient:  sts.NewFromConfig(cfg),
-		s3Client:   s3.NewFromConfig(cfg),
 	}, nil
 }
 
 // ConfigureIdPIAM creates a new IAM OIDC IdP in AWS.
 //
-// The Provider URL is Teleport's Public Address or the S3 bucket.
+// The Provider URL is Teleport's Public Address.
```

Comment:
minor nit: 
```suggestion
// The provider URL is Teleport's public address.
```
I could see it was always written like that but we only capitalize product/company names abbreviations etc.